pulumi/grapl/__main__.py

```diff
@@ -118,36 +132,37 @@ def main() -> None:
     )
     pulumi.export("ux-bucket", ux_bucket.bucket)
 
```

Comment:
apologies, i too refactored this whole section :(

Reply:
Meh